user_docs/en/userGuide.t2t

```diff
@@ -2925,6 +2926,30 @@ If you install an add-on with paid components and change your mind about using i
 The Add-on Store is accessed from the Tools submenu of the NVDA menu.
 To access the Add-on Store from anywhere, assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+++ Note on security and privacy when using Add-ons ++[AddonSecurityAndPrivacy]
+Installing add-ons in NVDA leads to integration of external code into NVDA's functionality in order to enhance NVDA or make new features possible.
+Add-ons can also use external libraries and third party services to serve the purpose and provide the features for which they have been developed. They can be developed by every person or company.
+
+Automated code checks for add-ons happen when they are submitted to NVDA’s Add-on Store. This means e.g.  they will undergo  a basic automated check for vulnerabilities by security analysis tools such as codeQL.
+There isn't currently any formal review process for add-ons performed by a human.
+However, many add-ons have discussions areas where users can exchange feedback. The [community review area #AddonStoreReviews] can be accessed via the actions menu of the add-on.
+
+Installed Add-ons or extensions (not only in NVDA) might in general introduce security and/or privacy vulnerabilities, depending on the permissions they need and actions they perform in order to provide the desired functionality.
+Risks can be e.g.
+- Insecure network connections
+- Files stored with insecure file permissions or in an unprotected location
+- Sensitive information written to an easily available log file
+- Vulnerabilities in third-party libraries
+- Cryptographic vulnerabilities, and more.

```

Comment:
"and more" is already implied. It does not help the case to say "and more", "etc.", or similar when giving unbounded examples.

Also, while I do not say it shouldn't be listed, I will point out that NVDA itself is already subject to most of these.

```suggestion
- Cryptographic vulnerabilities
```

Reply:
NVDA is still officially reviewed by an organization and its own project documentation is already very comprehensive, this will mittigate many vulnerabilities and there is a security whistleblowing environment to report security vulnerabilities already. Ofcourse we can also take a note on NVDA itself in the userguite, maybe linking to the security policy and SLA. But this is out of scope for this particular PR.